src/main/java/lotto/domain/WinningNumbers.java

```diff
@@ -0,0 +1,38 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class WinningNumbers {
+    private final Lotto lotto;
+    private final LottoNumber bonusNumber;
+
+    public WinningNumbers(Lotto lotto, LottoNumber bonusNumber) {
+        validateDuplicate(lotto, bonusNumber);
+        this.lotto = lotto;
+        this.bonusNumber = bonusNumber;
+    }
```

Comment:
```suggestion
    public WinningNumbers(List<Integer> lottoNumbers, int bonusNumber) {
        this(new Lotto(lottoNumbers), new LottoNumber(bonusNumber);
    }
```
이와 같은 부 생성자를 추가하면 WinningNumbers을 사용하는 개발자 입장에서 객체를 생성할 때 편리함을 얻을 수 있지 않을까?
객체를 생성하기 위한 여러 타입을 가지는 생성자를 추가하는 방식으로 접근해 보면 어떨까?